### ISSUE TITLE
Faveo looking for bugsnag in incorrect directory?

### ISSUE THREAD

**Issue by tcreek** [ladybirdweb/faveo-helpdesk#2489]
just downloaded most current community edition

After seeing a blank screen, I am seeing the following error in the httpd error_log.

>  PHP Fatal error:  Uncaught Error: Class 'Bugsnag' not found in /var/www/html/                              faveo/app/Exceptions/Handler.php:50\nStack trace:\n#0 /var/www/html/faveo/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(81): App\\E                              xceptions\\Handler->report(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#1 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExcepti                              ons->handleException(Object(Symfony\\Component\\Debug\\Exception\\FatalThrowableError))\n#2 {main}\n  thrown in /var/www/html/faveo/app/Exceptions/Handler.php on line                               50

Doing a search of bugsnag reveals it is in  ../faveo/vendor/bugsnag/ not ../faveo/vendor/laravel 

Am I reading that correctly?

**Comment by mverma16** [comment]
Hi @tcreek the issue might be due to different reasons. Please try to open `your_faveo_url/probe.php` and see if it shows and error. Other troubleshoots steps are as below
- Make sure the directory permission are according to [Laravel requirement](https://laravel.com/docs/5.8/#configuration)
- Your server is running PHP7.1 or greater version

**Comment by tcreek** [comment]
Before even installing, I checked the install with the Faveo Probe.  Everything checked out okay.

However, I had it installed on a virtual machine. That machine for some reason would no longer start up, so I started all over again.

It is fine, so far.

Thank you for responding. I can see how this got lost among the flood of issues that the other has been constantly flooding in this section. 

If I have any more issues during this setup , I will respond again.

Thank you again
